src/main.cpp

```diff
@@ -4119,7 +4119,9 @@ bool ContextualCheckBlock(const CBlock &block, CValidationState &state, CBlockIn
 
     // BUIP055 enforce that the fork block is > 1MB
```

Comment:
sorry to few caffeine, you are indeed right.
Maybe we should change BUIP055 into BUIP055/UAHF, cause we are actually following the UAHF spec.